### PR TITLE
anytopnm: add page

### PR DESCRIPTION
**Pull request by WillOfHeaven**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
 
Add commands to #10887

### CODE REVIEW

File: pages/common/anytopnm.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# anytopnm
+
+> Converts an arbitrary type of image file to common image formats.
+> More information: <https://netpbm.sourceforge.net/doc/anytopnm.html>.
+
+- Convert an input image to PBM, PGM, or PPM format irrespective of the input type:
+  
+`anotopnm`
```
Comment (pixelcmtd) [inline pages/common/anytopnm.md:8]:
```suggestion
`anytopnm {{path/to/input}} > {{path/to/output.pnm}}`
```

File: pages/common/anytopnm.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Print the version of the netpbm package that contains the anytopnm program:
```
Comment (pixelcmtd) [inline pages/common/anytopnm.md:10]: This has not been standardized _yet_, for the time being I'd suggest this:

```suggestion
- Display version:
```

**Comment by WillOfHeaven** [comment]
@pixelcmtd  I had a question : Why is it showing in the PR list that only 4 out of 5 task's have been completed.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**